cairo/src/font/font_face.rs

```diff
@@ -64,14 +69,16 @@ impl FontFace {
     /// `FontFace` is dropped.
     #[cfg(any(feature = "freetype", feature = "dox"))]
     #[doc(alias = "cairo_ft_font_face_create_for_ft_face")]
-    pub unsafe fn create_from_ft(
-        face: freetype_crate::freetype::FT_Face,
-    ) -> Result<FontFace, Error> {
-        let font_face = FontFace::from_raw_full(ffi::cairo_ft_font_face_create_for_ft_face(
-            face as *mut _,
-            0,
-        ));
-        let status = ffi::cairo_font_face_status(font_face.to_raw_none());
+    pub fn create_from_ft(face: &freetype::face::Face) -> Result<FontFace, Error> {
+        let mut face = face.clone();
+        let font_face = unsafe {
```

Comment:
Can you add a safety comment here? I missed the `.clone()` call at first and was worried about potential use-after-frees

Reply:
Better would be to use `// SAFETY: blablabla` but this also works :)